src/vaishnaviVJun20_arrayExamples/IndexOfEx.java

```diff
@@ -0,0 +1,26 @@
+package vaishnaviVJun20_arrayExamples;
+
+public class IndexOfEx {
+	
+	void printFirstAndLastIndex(String s) {
+		for(int index=0;index<s.length();index++) {
+			char ch = s.charAt(index);
+			int count=0;
+			if(index == s.indexOf(ch)) {
+				for(int i=0;i<s.length();i++) {
+					if(ch== s.charAt(i))
+						count++;				
+				}
```

Comment:
You have increased the complexity in code, Using one for loop and one if block you can achieve the output. 

Reply:
Thank you for suggestion, Changes done.